auth/auth_test.go

```diff
@@ -71,6 +71,18 @@ func TestAuthSnippets(t *testing.T) {
 		t.Errorf("getIdTokenFromServiceAccount got %q, want %q", got, want)
 	}
 
+	buf.Reset()
+	want = "Generated OAuth2 token"
+	impersonatedServiceAccount := "name@project.service.gserviceaccount.com"
```

Comment:
Needs to be changed to the service account name set up by the reviewer (@telpirion )

Reply:
@grayside is now reviewer

Reply:
Fixed in https://github.com/GoogleCloudPlatform/golang-samples/pull/3450/files/7c7e0309247a108b71e497c324335dc8e136516e..133e131986f263fae31d327a9f22b08e6c2b3467